fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -162,7 +162,7 @@ def create_report1(
     reports: List[Dict] = []
     no_result_keywords: List[str] = []
     for category, keywords in category_keywords.items():
-        for k in keywords:
+        for k in keywords:                
```

Comment:
some extra tabs / spaces got in here. NP, wouldn't cause an error anyway; i'll remove them